PVAO.Web/Views/Home/Index.cshtml

```diff
@@ -1,23 +1,27 @@
 ﻿@{
     ViewBag.Title = "Home Page";
 }
+@section styles {
+    <style>
+        h1 {
+            text-align: center;
+            color: #543fff;
+        }
+    </style>
```

Comment:
Can we do this externally? Add it to "Content/css/dasboard.css" for a lot cleaner code.